pkg/importer/avro/avro_util.arrai

```diff
@@ -0,0 +1,143 @@
+# Utilities
+
+let getAnnotationS = \item \name
+    let annotation = item(name)? .s:"";
+    let annotationTxt = 
+        cond annotation {
+            "":"", 
+            _: $'${cond name {'doc':'description', _:name}}="${annotation}"'
+        };
+    annotationTxt
+;
+
+# Pretty annotation array
+let prettyAnnotations = \annotationArray
+    let str = $'${let a = annotationArray where .@item != "" rank (:.@); a >>> \i \item item::, }';
+    cond str {'':$'', _:$'[${str}]'}
+;
+
+# Transform Avro primitive types to Sysl ones
+let transformPrimitiveType = \type
+    cond type {
+        'null': 'null',
+        'boolean': 'bool',
+        'int': 'Int32',
+        'long': 'Int64',
+        'float': 'Float32',
+        'double': 'Float64',
+        'bytes': "bytes",
+        'string': 'string',
+        _: 'NonPrimitiveType'              
+    }
+;
+
+let transformLogicalTypes = \type
+    cond type('logicalType').s {
+        'decimal': $'decimal(${type('precision')}.${type('scale')})',
+        'uuid': 'string',
+        'date': 'date',
+        'time-millis': 'uint32',
+        'time-micros': 'uint64',
+        'timestamp-millis': 'datetime',
+        'timestamp-micros': 'datetime',
+        'local-timestamp-millis': 'datetime',
+        'local-timestamp-micros': 'datetime',
+        'duration': 'bytes',
+        _: 'NonLogicalType'  
+    }
+;
+
+let isFirstLetterUppercase = \str
+    let letter = str where .@ = 0;
+    //str.upper(letter) = letter
+;
+
+let correctName = \name
+    cond transformPrimitiveType(name) {
```

Comment:
Nit: Group this function with `transformPrimitiveType`, since it appears to be the only user.